.copier-answers.yml

```diff
@@ -1,17 +1,25 @@
 # Changes here will be overwritten by Copier
-_commit: v1.4.6
-_src_path: gh:lincc-frameworks/python-project-template
+_commit: v1.5.0-1-g65bbf6e
+_src_path: ./python-project-template/
```

Comment:
Is it ok that this isn't from github anymore?

Reply:
It's a little dangerous to adjust this manually, and I ran the copier update with a local version of the template, since there are some un-merged fixes.

But I'm feeling lucky today, so I went ahead and manually tweaked to what the file will look like soon.